plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/ILibraryRepositoryService.java

```diff
@@ -26,8 +26,10 @@
 
   /**
    * @return a path that points to a local file corresponding to the artifact described by <code>mavenCoordinates</code>
+   * @throws LibraryRepositoryServiceException if the resolution of the artifact defined by
+   * <code>mavenCoordinates</code> fails.
    */
-  IPath getJarLocation(MavenCoordinates mavenCoordinates);
+  IPath getJarLocation(MavenCoordinates mavenCoordinates) throws LibraryRepositoryServiceException;
```

Comment:
And we'll need a Javadoc location too — Objectify has its javadoc jar up on [Maven Central](http://repo1.maven.org/maven2/com/googlecode/objectify/objectify/5.1.13/)